packages/ui-extensions-server-kit/src/components/ActionSet/actions/RefreshAction/RefreshAction.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+// import {useI18n} from '@shopify/react-i18n';
```

Comment:
Can we remove this until we're ready to use it?

Reply:
I'll remove it in a separate commit